packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -105,7 +105,7 @@ abstract class LambdaAuthorizer extends Authorizer implements IAuthorizer {
       this.role.attachInlinePolicy(new iam.Policy(this, 'authorizerInvokePolicy', {
         statements: [
           new iam.PolicyStatement({
-            resources: [this.handler.functionArn],
+            resources: [this.handler.functionArn], // TODO: maybe this one too
```

Comment:
This one didn't have a `// TODO` on it before but I feel like it's reasonable to change this one also. I didn't do it yet just in case it was omitted for a reason.

Reply:
Good catch, you're right. This one needs to be updated too.